app/views/layouts/darkswarm.html.haml

```diff
@@ -11,6 +11,7 @@
     - else
       = favicon_link_tag "/favicon-staging.ico"
     %link{href: "https://fonts.googleapis.com/css?family=Roboto:400,300italic,400italic,300,700,700italic|Oswald:300,400,700", rel: "stylesheet", type: "text/css"}
+    %link{href: "https://opensource.keycdn.com/fontawesome/4.7.0/font-awesome.min.css", rel: "stylesheet"}
```

Comment:
Now I see that `fa` is short for Font Awesome. Okay, is this just for the globe symbol? Do we need to discuss introducing another external source? This triggers a few more requests. Maybe it would be better to copy only what we need? Opinions?

Reply:
I've rebuilt the OFN icon font with the additional globe icon using IcoMoon and removed FontAwesome from the frontend.